### DISCUSSION TITLE
Tooltip

### DISCUSSION THREAD

**Discussion by jassirahmed** [upsetjs/upsetjs#133] [Q&A]
I Want custom tooltip for render my data please i need help for this

**Comment by sgratzl** [comment]
you can disable the native tooltips via this option: https://github.com/upsetjs/upsetjs/blob/22c8c303fe4455b059940510bd4cb06d4211b033/packages/react/src/interfaces.ts#LL590C3-L590C11

then you can use https://github.com/upsetjs/upsetjs/blob/22c8c303fe4455b059940510bd4cb06d4211b033/packages/react/src/interfaces.ts#L382-L390 to listen to events when the user is over an set
